tensor/datafs/file.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package datafs
+
+import (
+	"bytes"
+	"io"
+	"io/fs"
+)
+
+// File represents a data item for reading, as fs.File.
+// All io functionality is handled by bytes.Reader
+type File struct {
```

Comment:
`datafs.File` vs `datafs.Data` seems confusing.

Reply:
this is the fs API.